src/services/nul-api.js

```diff
@@ -22,6 +22,7 @@ export function login() {
 
 export function logout() {
   localStorage.removeItem(loginKey);
+  localStorage.removeItem('currentUser');
```

Comment:
I had reasons for leaving `currentUser` alone on logout – I think because I wanted it to remember the user when they logged back in. But maybe the SSO Token is sufficient (and a better idea). I just want to make sure there's a way to be explicitly anonymous (logged out) even if the browser contains a valid SSO token cookie.